### PR TITLE
included simple energy usage and 

### PR DESCRIPTION
**Pull request by Storas008**
included simple energy and cost calculation algorithm

### CODE REVIEW

File: Services/EnergyCostAlgorithm.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+public class EnergyCostAlgorithm
+{
+    private static double totalEnergy = 0;
+    private static double totalEnergyCost = 0;
+    private static float costPerkWh = 0;
+
+    // powerScale is a proportion of power used for the specified time; scales 1 to 10; ex.: medium heat, 3/10 of power
+    private void electricPower (int appliencePower, short powerScale, int timeInMin){
+        totalEnergy += appliencePower * powerScale * timeInMin / 60_000;
```
Comment (Jurshenas) [inline Services/EnergyCostAlgorithm.cs:12]: If we have a 1000W appliance that's always working at full power (10) then we should get 1kWh if it works for 60 mins.
With this calculation we get 1000W * 10 * 60min / 60_000 = 10, which is strange.
In my opinion powerScale should be a float, which represents how much power the appliance is using. For example a 1000W appliance working at 3/10 powerScale would be basically a 300W appliance. With this change everything seems to work perfectly.

**Review by Jurshenas — CHANGES_REQUESTED** [review]
Added comments, please look into it

File: Services/EnergyCostAlgorithm.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private double gasPowerCost (int appliancePower, short powerScale, int timeInMin, float costPerLitre){
+        return 0;
+    }
+
+    // getters n setters:
+    public double getTotalEnergy(){ return totalEnergy; }
+
+    public double getTotalEnergyCost() { return totalEnergyCost ;}
+
+    public void setCostPerkWh(float cost){ costPerkWh = cost; }
```
Comment (Azuolas3) [inline Services/EnergyCostAlgorithm.cs:29]: Please look into C# Properties, it makes writing getters and setters very simple, 
for ex. when you declare variables, you can write simply `public TotalEnergy { get; set; }` and C# will take care of basically everything

File: Services/EnergyCostAlgorithm.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
+namespace Ultra_Saver;
+
+public class EnergyCostAlgorithm
+{
+    private static double totalEnergy = 0;
+    private static double totalEnergyCost = 0;
+    private static float costPerkWh = 0;
```
Comment (Azuolas3) [inline Services/EnergyCostAlgorithm.cs:8]: Not sure if these variables should be static, we might have problems if there will be multiple instances of this object (not sure though)
Also, look into properties, further explanation below

File: Services/EnergyCostAlgorithm.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    private void electricPower (int appliancePower, short powerScale, int timeInMin){
+        totalEnergy += appliancePower * powerScale * timeInMin / 60_000;
+        totalEnergyCost = Math.Round (totalEnergy * costPerkWh, 2);
+    }
+
+    private double gasPower (int appliancePower, short powerScale, int timeInMin){
+        return 0;
+    }
+
+    private double gasPowerCost (int appliancePower, short powerScale, int timeInMin, float costPerLitre){
```
Comment (Azuolas3) [inline Services/EnergyCostAlgorithm.cs:20]: Minor problem, but C# methods are usually written in PascalCase, which means that the first letter is capitalized as well
ElectricPower(), GasPower(), GasPowerCost(), etc.

File: Services/EnergyCostAlgorithm.cs (hunk 1 above)
Comment (Storas008) [reply Services/EnergyCostAlgorithm.cs]: thank you for noticing this mistake.

File: Services/EnergyCostAlgorithm.cs (hunk 3 above)
Comment (Storas008) [reply Services/EnergyCostAlgorithm.cs:8]: This probably is going to create problems

File: Services/EnergyCostAlgorithm.cs (hunk 2 above)
Comment (Storas008) [reply Services/EnergyCostAlgorithm.cs]: ok

**Pull request merged**